### PR TITLE
New patch release of Golang: v1.19.13

### PR DESCRIPTION
**Pull request by rcrozean**
Fix for CVE-2023-45283 & CVE-2023-45284

Backported fix from: https://github.com/golang/go/commit/46fb78168596f7ce8834f528bb0eb9555c08bcae

Additional cherry-picks:
  https://github.com/golang/go/commit/45b98bfb793923c539f9a959c3047d2e5fe2ebf0 - add prepend and additional windows checks
  https://github.com/golang/go/commit/6d0bf438e302afcb0db5422ea2da59d1995e08c1 - adds isLocal 

Update EKS Go Patch Version: v1.19.13-13
SPEC FILE STILL NEEDS THE '%changelog' UPDATED
PLEASE UPDATE WITH THE FOLLOWING FORMAT
```
* Wed Sep 06 2023 Cameron Rozean <rcrozean@amazon.com> - 1.20.8-1
- Bump tracking patch version to 1.20.8 from 1.20.7
```

### CODE REVIEW

File: projects/golang/go/1.19/README.md [hunk 1]
```diff
@@ -24,4 +24,4 @@ Tracking Tag: `go1.19.13`
 The patches in `./patches` include relevant utility fixes for go `1.19`.
 
 ### Spec
-The RPM spec file in `./rpmbuild/SPECS` is sourced from the go 1.19 SRPM available on Fedora, and modified to include the relevant patches and build the `go1.19.13` source.
+The RPM spec file in `./rpmbuild/SPECS` is sourced from the go 1.19 SRPM available on Fedora, and modified to include the relevant patches and build the `go1.19.13` source."
```
Comment (jiayiwang7) [inline projects/golang/go/1.19/README.md:27]: why do we need this `"`

Comment (zafs23) [inline projects/golang/go/1.19/README.md:27]: there is an extra `"` at the end of "spec" section in the readme.

**Pull request merged**